doc/metrics.md

```diff
@@ -151,6 +155,12 @@ scrape_configs:
       #      is not covered by the certificate (not in the SAN list)
       server_name: 'foo'
 ```
+```{note}
+The `scrape_interval` is assumed to be 15s by the Grafana Prometheus data source by default.
+
+When using a different `scrape_interval` value, change it too in the Grafana Prometheus data source configuration.
+Otherwise the Grafana `$__rate_interval` value will be calculated incorrectly and possibly cause a `no data` response in queries using it.
+```
 
 ````{note}
```

Comment:
We shouldn't have two consecutive notes - please change to one note with a bullet list for the different points.

Reply:
> We shouldn't have two consecutive notes - please change to one note with a bullet list for the different points.

Limited experience on formatting, this is the closest I get.